ignition-server/controllers/local_ignitionprovider.go

```diff
@@ -192,6 +197,24 @@ func (p *LocalIgnitionProvider) GetPayload(ctx context.Context, releaseImage str
 	}
 
 	// Extract template files from the MCO image to the MCC input directory
+	isManifestListImage, err := registryclient.IsMultiArchManifestList(ctx, mcoImage, pullSecret)
```

Comment:
why name different the variable (isManifestListImage) that receives the content from IsMultiArchManifestList?

shouldn't it just be isMultiArchManifestList := IsMultiArchManifestList()

Reply:
I'm good with that change. It's been updated.